docs/env.example

```diff
@@ -0,0 +1,33 @@
+# This file is Free Software under the Apache-2.0 License
+# without warranty, see README.md and LICENSES/Apache-2.0.txt for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# SPDX-FileCopyrightText: 2024 German Federal Office for Information Security (BSI) <https://www.bsi.bund.de>
+#  Software-Engineering: 2024 Intevation GmbH <https://intevation.de>
+
+# Where to reach the application's keycloak-instance
+PUBLIC_KEYCLOAK_URL="http://localhost:8080"
+
+# The name of the keycloak-realm managing all ISDuBA accounts
+PUBLIC_KEYCLOAK_REALM="isduba"
+
+# The name of the client that can request the authentification of a user
```

Comment:
Since we have code in a folder named `client` and the application itself is a client I would specify here that a keycloak-client is meant (like it is already in the name of the variable).

Reply:
I specified keycloak-client, does this suffice?